packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
@gdoyle1 Here is the block of code to use at the top of the page for referencing the GIFs:

```
---
path: "/design-guidelines/styles/colors-for-charts"
---
import theSystem from './thesystem.gif';
import oneFamily from './onefamily.gif';
```